examples/live/generate_injections.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+
+import os, sys
+from pycbc.io import record
+import h5py
+from pycbc.inject import InjectionSet
+
+if os.path.exists('./test_inj.hdf'):
+    raise OSError("output-file already exists")
+
+# 2 injections 
+static_params = { 'f_lower': 18.0, 'f_ref': 20.0, \
```

Comment:
I suggest setting `f_ref` equal to `f_lower` as not all approximants like them different.